rabe-icecast-theme.spec

```diff
@@ -0,0 +1,64 @@
+#
+# spec file for package rabe-icecast-theme
+#
+# Copyright (c) 2017 Radio Bern RaBe
+#                    http://www.rabe.ch
+#
+# This program is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public 
+# License as published  by the Free Software Foundation, version
+# 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License  along with this program.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Please submit enhancements, bugfixes or comments via GitHub:
+# https://github.com/radiorabe/rabe-icecast-theme
+#
+
+Name:           rabe-icecast-theme
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        Radio Bern RaBe Icecast web theme
+
+License:        AGPL
+BuildArch:      noarch
+URL:            https://github.com/radiorabe/%{name}
+Source0:        https://github.com/radiorabe/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+
+%description
+Icecast streaming server web layout (CSS, HTML and images) for Radio Bern RaBe.
+
+
+%prep
+%setup -q -n %{name}-%{version}
+
+
+%build
+
+
+%install
+rm -rf $RPM_BUILD_ROOT
```

Comment:
I *think* these aren't needed anymore (ie. they where removed from the Fedora packaging guidelines). I may be wrong though.

Reply:
This was automatically added by `rpmdev-newspec` shipped with CentOS 7. Although, it would probably work without it, I see no harm in having it there (especially as `rpmdev-newspec`'s upstream [considers it as the correct way](https://pagure.io/rpmdevtools/issue/25#comment-50986)). See also the [comment 4 on #807857](https://bugzilla.redhat.com/show_bug.cgi?id=807857#c4)

Reply:
The packaging guidelines has long since removed this recommendation. rpmdev-newspec still has this since it needs to create templates that support EPEL5 and earlier.

Nowadays the package guidelines [clearly state](https://fedoraproject.org/wiki/Packaging:Guidelines?rd=PackagingGuidelines#Tags_and_Sections) the following:

> * The contents of the buildroot SHOULD NOT be removed in the first line of %install.

Given that the comment you linked is from 2012 I consider the package guidelines to win as they have been clarified by adding a explicit SHOULD NOT.

Reply:
> rpmdev-newspec still has this since it needs to create templates that support EPEL5 and earlier.

so even if  `rpmdev-newspec` is doing it, which is shipped by the same distro as the package is supposed to be installed on, you consider it to be wrong?

> I consider the package guidelines to win as they have been clarified by adding a explicit SHOULD NOT.

Yeah, which is a) the guideline for Fedora which comes in a much more modern fashion than CentOS and b) written as *SHOULD NOT* and not *MUST NOT*. But anyway, I couldn't care less and don't plan on spending more time at arguing about nitpicking details on a SPEC file which no one else except us would ever use. => Removed and pushed.